hooks/_common.sh

```diff
@@ -445,6 +445,46 @@ function common::colorify {
   echo -e "${COLOR}${TEXT}${RESET}" >&2
 }
 
+#######################################################################
+# Set terraform binary path
+# Allows user to set the path to custom Terraform or OpenTofu binaries
+# Outputs:
+#   If failed - exit 1 with error message about missing Terraform/Opentofu binary
+function common::get_tf_binary {
+  local hook_config_tf_binary
+
+  for config in "${HOOK_CONFIG[@]}"; do
+    if [[ $config == tf_binary=* ]]; then
+      hook_config_tf_binary="${config#*=}"
+      break
+    fi
+  done
+
+  # direct hook config, has the highest priority
```

Comment:
Not essential, though maybe "precedence" is a bit more correct in this context? 🤔 
```suggestion
  # direct hook config, has the highest precedence
```